src/FSharpVSPowerTools.Logic/QuickInfoMargin.fs

```diff
@@ -136,7 +136,7 @@ type QuickInfoMargin (textDocument: ITextDocument,
             |> Async.StartInThreadPoolSafe
         | None -> updateQuickInfo (None, None)
 
-    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 1000us, updateAtCaretPosition)
+    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 200us, updateAtCaretPosition)
```

Comment:
With 1s delay it feels like a slow feature:

![1](https://cloud.githubusercontent.com/assets/873919/10539222/c210c88a-7406-11e5-9c20-a7aa7b178662.gif)